metabase/metabase
# Refactor AppBar

Epic https://github.com/metabase/metabase/issues/22826

Refactor `AppBar` to get remove `position: absolute`, so it would be easier to add `QuestionLineage` later. Please note there is an (already existing) issue with `CollectionBreadcrumbs` wrapping, to be fixed separately.

How to test:
- Make sure the sidebar bar can be opened by hovering on the logo, as before
- Make sure the collection path is shown when opening a question and closing the sidebar
